core/src/br/cefetmg/games/logic/chooser/GameSequencer.java

```diff
@@ -64,15 +64,15 @@ public boolean hasNextGame() {
     }
 
     private void determineGameSequence() {
-        ArrayList<Integer> indexList = new ArrayList<Integer>();
-        
-        for (int i = 0; i < numberOfGames; i++)
-            indexList.add(i);
+        int lastIndex = -1; // garante que o primeiro sorteio não será igual a lastIndex
         
         for (int i = 0; i < numberOfGames; i++) {
-            int index = MathUtils.random(indexList.size() - 1);
-            indexSequence[i] = indexList.get(index);
-            indexList.remove(index);
+            int index = MathUtils.random(availableGames.size() - 1);
+            if (index != lastIndex) { // garante que o último ínidce sorteado não irá repetir
+                indexSequence[i] = index;
+                lastIndex = (availableGames.size() == 1) ? -1 : index;
+            } else
```

Comment:
@jaguarsurf e @sarahrms, em termos de Engenharia de Software, é importante sempre usar chaves para criar um bloco mesmo quando temos apenas uma instrução dentro de um if/else/for/while. Vejam esta [resposta no StackOverflow com os motivos](https://stackoverflow.com/a/12193202/1783793).